spec/models/item_status_spec.rb

```diff
@@ -3,7 +3,7 @@
 def update_symphony_data_and_save(request, symphony_data)
   r = Request.find(request.id)
   r.symphony_response_data = symphony_data
-  r.save!
+  r.save
```

Comment:
It's to avoid validation, which can be a bit timely (I believe due to getting current location from the SW API).